deprecations/README.md

```diff
@@ -18,37 +18,37 @@ The table columns have the following meanings:
-| `customers/getAllByIds` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
-| `customers/getAllByEmails` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
-| `customers/getAllByName` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
+| [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2017-04-12)<br>(ver 2017-04-12) | Replaced by [Get all reservations \(ver 2023-06-06\)](../operations/reservations.md#get-all-reservations-ver-2023-06-06) | 23 Jun 2023 | 23 Jun 2024 |
+| [Get all reservation items](../operations/reservations.md#get-all-reservation-items) | Replaced by [Get all order items](../operations/orderitems.md#get-all-order-items) | 23 Jun 2023 | 23 Jun 2024 |
+| [Get all accounting items](../operations/accountingitems.md#get-all-accounting-items) | Replaced by [Get all payments](../operations/payments.md#get-all-payments) and [Get all order items](../operations/orderitems.md#get-all-order-items) | 19 Apr 2023 | 19 Apr 2024 |
+| [Merge customers](../operations/customers.md#merge-customers) | Replaced by [Merge accounts](../operations/accounts.md#merge-accounts) | 01 Dec 2022 | 01 Dec 2023 |
+| `Get all companies by name` | Replaced by [Get all companies](../operations/companies.md#get-all-companies) | 01 Jan 2020 | - |
+| `Get all bills by Id` | Replaced by [Get all bills](../operations/bills.md#get-all-bills) | 01 Jan 2020 | - |
+| `Get all bills by customer` | Replaced by [Get all bills](../operations/bills.md#get-all-bills) | 01 Jan 2020 | - |
+| `Get all closed bills` | Replaced by [Get all bills](../operations/bills.md#get-all-bills) | 01 Jan 2020 | - |
+| `Get all credit cards by Id` | Replaced by [Get all credit cards](../operations/creditcards.md#get-all-credit-cards) | 01 Jan 2020 | - |
+| `Get all credit cards by customer` | Replaced by [Get all credit cards](../operations/creditcards.md#get-all-credit-cards) | 01 Jan 2020 | - |
+| `Get all reservations by Id` | Replaced by [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2017-04-12) | 01 Jan 2020 | - |
+| `Get all reservations by customer` | Replaced by [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2017-04-12) | 01 Jan 2020 | - |
+| `Get all reservations by numbers` | Replaced by [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2017-04-12) | 01 Jan 2020 | - |
+| `Get all customers by Ids` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
+| `Get all customers by emails` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
+| `Get all customers by name` | Replaced by [Get all customers](../operations/customers.md#get-all-customers) | 01 Jan 2020 | - |
```

Comment:
Imho we should put links to all these operations as well, to be more consistent, since you're touching the naming 😁

Reply:
@VBuben these are discontinued operations that are no longer in the documentation!